package.json

```diff
@@ -40,7 +40,8 @@
     "@cypress/skip-test": "^2.6.1",
     "@opensearch-dashboards-test/opensearch-dashboards-test-library": "https://github.com/opensearch-project/opensearch-dashboards-test-library/archive/refs/tags/1.0.5.tar.gz",
     "brace": "^0.11.1",
-    "prettier": "^2.5.1"
+    "prettier": "^2.5.1",
+    "yarn": "^1.22.19"
```

Comment:
Is this necessary?

Reply:
i think i did this accidentally when i was setting up cypress. It is not necessary

Reply:
Can you remove it then?

Reply:
removed